### DISCUSSION TITLE
standard_init_linux.go:228: exec user process caused: exec format error

### DISCUSSION THREAD

**Discussion by nitrup** [OlivierGaland/CashFactory#16] [Q&A]
Hi all,

I used Unbuntu 20.04 LTS

On the container bellow I have the error on log : standard_init_linux.go:228: exec user process caused: exec format error

-[cashfactory_Packetstream_PsClient_1]
-[cashfactory_Peer2profit_1]
-[cashfactory_Honeygain_1]

I follow the installation description all is running but on log this error normal ?

Thanks

**Comment by OlivierGaland** [comment]
Hello,

I got the same error when trying to launch some container on raspberry (arm cpu), I believe because there is no docker image for this platform. But not sure if it is a good hint in your case. What is your setup ?

This error is most likely to cause the app not running for containers having this error (so peer2profit, packetstream and honeygain)

**Comment by nitrup** [comment]
Hello,

I use Rpi 2 and 3 and one Freebox Delta with Quad-Core ARMv8

**Comment by Sigmoidas** [comment]
The following tutorial allowed me to figure out how to run Honeygain Docker image on Raspberry Pi 3 (RPi 3) hardware: https://www.reddit.com/r/Honeygain/comments/tj8vfa/how_to_install_honeygain_on_a_raspberry_pi_with/ 

The key is running Honeygain Docker image in an "abstraction layer", and utilizing the `--platform linux/amd64` flag:
`sudo docker run -d --restart always --platform linux/amd64 --name honeygain honeygain/honeygain -tou-accept -email MAIL@mail.nu -pass PASSWORD123 -device raspberry_pi
`
